osgp/protocol-adapter-iec60870/README.md

```diff
@@ -10,3 +10,64 @@ Grid eXchange Fabric detailed documentation:
 
 Grid eXchange Fabric issue tracker:
 * [Grid eXchange Fabric Jira](https://smartsocietyservices.atlassian.net/projects/OC/issues/)
+
+## Add 104 devices to the database
+
+...
+-- osgp_core
+
+INSERT INTO device
+    (creation_time, modification_time, version, device_identification, device_type, is_activated,
+     protocol_info_id, network_address, in_maintenance, gateway_device_id,
+     device_model, device_lifecycle_status)
+  VALUES
+    (current_timestamp, current_timestamp, 0, 'GATEWAY_1', null, true,
+        (SELECT id FROM protocol_info WHERE protocol = '60870-5-104' AND protocol_version = '1.0'),
+        '127.0.0.1', false, null,
+        (SELECT id FROM device_model WHERE model_code = 'Test' AND manufacturer_id = (SELECT id FROM manufacturer WHERE code = 'Test')),
```

Comment:
Not sure if we already have a proper device model available which could be used for light measurement gateway, light measurement devices or both...